content/docs/releases/upgrading.mdx

```diff
@@ -13,6 +13,24 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## Since 0.22.0
+
+### New
+
+#### Hosted Authenticate Service
+
+The [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service.md) replaces the need for an explicit authenticate service URL and hosted identity provider. Users can now get started without including their own identity provider, or authenticate service.
+
+The [Self-Hosted Authenticate Service](/docs/capabilities/self-hosted-authenticate-service), which requires a self-hosted authenticate service URL and identity provider, is still supported for users that prefer to host these services themselves.
+
+#### Wildcard From Routes
+
+[Wildcard From Routes](/docs/reference/routes/from#wildcard-from-routes) support allows you to define multiple near-identical external `from` routes that point to the same internal `to` route. Wildcard From Routes is a Beta feature.
```

Comment:
> Wildcard From Routes

Is awkwardly written. The feature allows wildcard syntax (e.g. `*`) in the `From` route configuration option.